Jint/Native/Function/FunctionInstance.cs

```diff
@@ -15,23 +15,35 @@ public abstract class FunctionInstance : ObjectInstance, ICallable
         private const string PropertyNameLength = "length";
         private const int PropertyNameLengthLength = 6;
         protected PropertyDescriptor _length;
+
+        private const string PropertyNameName = "name";
+        private const int PropertyNameNameLength = 4;
```

Comment:
I assume for performance reasons. Isn't the `==` operator on string cleaver enough to already apply this?

Reply:
I measured it with BDN at it showed to be a bit faster when we don't need a null check before we do length check, so a real micro-optimization that might be something to remove at some point.